src/sage/categories/coxeter_groups.py

```diff
@@ -1876,40 +1883,40 @@ def absolute_le(self, other):
 
             EXAMPLES::
 
-                sage: W = WeylGroup(["A", 3])
-                sage: s = W.simple_reflections()
-                sage: w0 = s[1]
-                sage: w1 = s[1]*s[2]*s[3]
-                sage: w0.absolute_le(w1)
+                sage: W = WeylGroup(["A", 3])                                           # optional - sage.combinat sage.groups
+                sage: s = W.simple_reflections()                                        # optional - sage.combinat sage.groups
+                sage: w0 = s[1]                                                         # optional - sage.combinat sage.groups
+                sage: w1 = s[1]*s[2]*s[3]                                               # optional - sage.combinat sage.groups
+                sage: w0.absolute_le(w1)                                                # optional - sage.combinat sage.groups
                 True
-                sage: w1.absolute_le(w0)
+                sage: w1.absolute_le(w0)                                                # optional - sage.combinat sage.groups
                 False
-                sage: w1.absolute_le(w1)
+                sage: w1.absolute_le(w1)                                                # optional - sage.combinat sage.groups
                 True
 
             TESTS:
 
             Check that this is independent of the implementation of the group, see :trac:`34799`::
 
-                sage: W1 = WeylGroup(['A',2])
-                sage: W2 = Permutations(3)
+                sage: W1 = WeylGroup(['A', 2])                                                                          # optional - sage.combinat sage.groups
+                sage: W2 = Permutations(3)                                                                              # optional - sage.combinat sage.groups
                 sage: P = lambda pi: W2(list(pi.to_permutation()))
-                sage: d1 = set((P(w1), P(w2)) for w1 in W1 for w2 in W1 if w1.absolute_le(w2))
-                sage: d2 = set((w1, w2) for w1 in W2 for w2 in W2 if w1.absolute_le(w2))
-                sage: d1 == d2
+                sage: d1 = set((P(w1), P(w2)) for w1 in W1 for w2 in W1 if w1.absolute_le(w2))                          # optional - sage.combinat sage.groups
+                sage: d2 = set((w1, w2) for w1 in W2 for w2 in W2 if w1.absolute_le(w2))                                # optional - sage.combinat sage.groups
+                sage: d1 == d2                                                                                          # optional - sage.combinat sage.groups
```

Comment:
```suggestion
                sage: d1 = set((P(w1), P(w2)) for w1 in W1 for w2 in W1                # optional - sage.combinat sage.groups
                ....:                         if w1.absolute_le(w2))
                sage: d2 = set((w1, w2) for w1 in W2 for w2 in W2                      # optional - sage.combinat sage.groups
                ....:                   if w1.absolute_le(w2))                         
                sage: d1 == d2                                                         # optional - sage.combinat sage.groups
```
Could also split the `for` in a second line.